### ISSUE TITLE
Source code organization - auto dl libraries

### ISSUE THREAD

**Issue by vindarel** [rmhsilva/lispmethods#2]
Hi again !

so this post is really helpful, thanks (http://lispmethods.com/libraries.html).

In the end, it is too bad you don't show solutions on how to automatically fetch dependencies without manually symlinking our projects to local-projects. I am *pretty sure* we can C-c C-k the .asd of our project, anywhere it might be, and quickload it.

What is `asdf:oos` ? You don't present `asdf:load-system` or `make` ? (https://gitlab.common-lisp.net/asdf/asdf/blob/master/doc/best_practices.md)

Indeed a `slime-load-project` would be great. What about opening an issue on Slime ? :)

About Quicklisp, it may be worth saying that it is different or that it goes further than pip in ensuring that all libraries work together in so-called dists… that are built once a month. That way one wouldn't be surprised and/or frustrated by this difference. (cf https://www.reddit.com/r/Common_Lisp/comments/6x7c85/why_do_we_have_to_wait_one_month_before_quicklisp/) And there's Qlot.

keep up the writing !

**Comment by rmhsilva** [comment]
@vindarel Thanks so much for the encouragement and suggestions! I need to get back to writing and improving lispmethods.com. Will do so soon.

**Comment by vindarel** [comment]
btw, I also used cl-arrows but I discovered more nice stuff in arrow-macros (https://github.com/hipeta/arrow-macros), like the `some->` ones (if a form returns nil, the macro returns nil instead of feeding it to the next form - sthg like monads).

**Comment by vindarel** [comment]
btw, I also used cl-arrows but I discovered more nice stuff in arrow-macros (https://github.com/hipeta/arrow-macros), like the `some->` ones (if a form returns nil, the macro returns nil instead of feeding it to the next form - sthg like monads).

**Comment by rmhsilva** [comment]
Oh nice, yep it has a few more. To be honest, that section of the article is quite opinionated anyway. I'm not sure if I should leave it in or not.

**Comment by vindarel** [comment]
yeah you can, I believe it's always nice to have links to libraries and to see an author's choice. You can also link to the https://github.com/CodyReichert/awesome-cl list :)
